app/src/forms/admin/routes.js

```diff
@@ -42,21 +195,193 @@ routes.get('/forms/:formId/versions/:formVersionId', async (req, res, next) => {
+ *   requestBody:
+ *    required: true
+ *    content:
+ *      application/json:
+ *        schema:
+ *          type: object
+ *          properties:
+ *            formId:
+ *              type: string
+ *              description: ID of the form
+ *              format: uuid
+ *              example: c6455376-382c-439d-a811-0381a012d696
+ *            userId:
+ *              type: string
+ *              format: uuid
+ *              description: ID of the User. This Id is from the User table in database.
+ *              required: true
+ *              example: 7cc11a0f-a7c9-4cbe-bd92-a2a22781ad9f
+ *            role:
+ *              type: string
+ *              description: Value must be 'owner'
+ *              default: owner
+ *              required: true
+ *              example: owner
+ *   responses:
+ *    '200':
+ *      description: Success
+ *      content:
+ *        application/json:
+ *          schema:
+ *            $ref: '#/components/responses/responseBody/AdminSetFormUserRoles'
+ *    '403':
+ *      $ref: '#/components/responses/Error/AccessDenied'
+ *    '5XX':
+ *      $ref: '#/components/responses/Error/UnExpected'
+ *    '404':
+ *        $ref: '#/components/responses/Error/ResourceNotFound'
+ *    '422':
+ *      description: 'Unprocessable Entity'
+ *      content:
+ *        application/json:
+ *          schema:
+ *            oneOf:
+ *              - $ref: '#/components/schemas/respError/UnprocessableEntityDBError'
+ *              - $ref: '#/components/schemas/respError/RoleRequiredValidationError'
+ *              - $ref: '#/components/schemas/respError/UserAndFormIdequiredError'
+ */
 routes.put('/forms/:formId/addUser', async (req, res, next) => {
   await controller.setFormUserRoles(req, res, next);
 });
 
 //
 // Users
 //
+
+/**
+ * @openapi
+ * /admin/users:
+ *  get:
+ *   tags:
+ *    - Admin
+ *   summary: Get the list of users
+ *   description: This endpoint will fetch all the users.
+ *   security:
+ *    - openId: []
+ *   responses:
+ *    '200':
+ *      description: Success
+ *      content:
+ *        application/json:
+ *          schema:
+ *            $ref: '#/components/responses/responseBody/AdminGetUsers'
+ *    '403':
+ *      $ref: '#/components/responses/Error/AccessDenied'
+ *    '5XX':
+ *      $ref: '#/components/responses/Error/UnExpected'
+ */
 routes.get('/users', async (req, res, next) => {
   await controller.getUsers(req, res, next);
 });
 
+/**
+ * @openapi
+ * /admin/users/{userId}:
+ *  get:
+ *    tags:
+ *      - Admin
+ *    summary: Get a user
+ *    description: This endpoint will fetch a single user using the userId passed through the endpoint path.
+ *    security:
+ *      - openId: []
+ *    parameters:
+ *      - in: path
+ *        name: formId
+ *        schema:
+ *          type: string
+ *          format: uuid
+ *        description: ID of the form.
+ *        required: true
+ *        example: c6455376-382c-439d-a811-0381a012d696
```

Comment:
This should be the example for userId, yes? With the userId `description` and and `example` values?